.github/workflows/pull_request.yml

```diff
@@ -1,6 +1,8 @@
 name: CI-PR
 
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
```

Comment:
slight weirdness here: a build is retriggered if someone edits the labels. I wanted to keep this to just 'feature preview' prs, so there's some discernment about which changes we actually expect someone to install & give feedback on. But we could also ditch that if frivolous retriggers are considered annoying